README.md

```
@@ -1 +1,32 @@
+# Currency_Convertor
+
+A SAMPLE project.
+
+## Features
+
+- Feature 1
+- Feature 2
+
+## Installation
+
+## Installation
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)